integration-tests/sql/README.adoc

```diff
@@ -0,0 +1,45 @@
+== SQL integration tests
+
+=== Default database type
+
+When the tests are executed without any special configuration, dev-service `H2` database is used (more details will follow).
+
+=== Dev-service databases
+
+As is described  in the https://quarkus.io/guides/datasource#dev-services[documentation], several database types could be started in dev-service mode.
+Running the tests against a database in dev-service mode could be achieved by addition of build property `SQL_JDBC_DB_KIND`. Example of usage:
+
+`mvn clean test -f integration-tests/sql/ -DSQL_JDBC_DB_KIND=postgresql`
+
+Following databases could be started in the dev-service mode:
+
+- Postgresql (container) - add `-DSQL_JDBC_DB_KIND=postgresql`
+- MySQL (container) - add `-DSQL_JDBC_DB_KIND=mysql`
+- MariaDB (container) - add `-DSQL_JDBC_DB_KIND=mariadb`
+- H2 (in-process) used by default
+- Apache Derby (in-process) - add `-DSQL_JDBC_DB_KIND=derby`
+- DB2 (container) (requires license acceptance) - add `-DSQL_JDBC_DB_KIND=db2`
+- MSSQL (container) (requires license acceptance) - add `-DSQL_JDBC_DB_KIND=mssql`
+
+For more information about dev-service mode, see https://quarkus.io/guides/datasource#dev-services[documentation].
```

Comment:
I wasn't aware there were that much kind of databases now with dev services.
I guess we can run the same tests only with these kinds of databases. Am I right?

Reply:
Yes,only dev services databases, which doesn't require a licence file to be present.

Reply:
done